src/python/evaluation/xlsx_run_tool.py

```diff
@@ -0,0 +1,170 @@
+from src.python.evaluation.evaluation_config import EvaluationConfig
+from src.python.review.application_config import LanguageVersion
+from src.python.review.common.file_system import create_file, new_temp_dir
+from src.python.review.common.subprocess_runner import run_in_subprocess
+from src.python.review.reviewers.perform_review import OutputFormat
+
+logger = logging.getLogger(__name__)
+
+
+def configure_arguments(parser: argparse.ArgumentParser, run_tool_arguments: enum.EnumMeta) -> NoReturn:
+    parser.add_argument('xlsx_file_path',
+                        type=lambda value: Path(value).absolute(),
+                        help='Local XLSX-file path. '
+                             'Your XLSX-file must include column-names: '
+                             f'"{EvaluationProcessNames.CODE.value}" and '
+                             f'"{EvaluationProcessNames.LANG.value}". Acceptable values for '
+                             f'"{EvaluationProcessNames.LANG.value}" column are: '
+                             f'{LanguageVersion.PYTHON_3.value}, {LanguageVersion.JAVA_8.value}, '
+                             f'{LanguageVersion.JAVA_11.value}, {LanguageVersion.KOTLIN.value}.')
+
+    parser.add_argument('-tp', '--tool-path',
+                        default=Path('src/python/review/run_tool.py').absolute(),
+                        type=lambda value: Path(value).absolute(),
+                        help='Path to script to run on files.')
+
+    parser.add_argument('-tr', '--traceback',
+                        help='If True, column with the full inspector feedback will be added '
+                             'to the output file with results.',
+                        action='store_true')
+
+    parser.add_argument('-ofp', '--output-folder-path',
+                        help='An absolute path to the folder where file with evaluation results'
+                             'will be stored.'
+                             'Default is the path to a directory, where is the folder with xlsx_file.',
+                        # if None default path will be specified based on xlsx_file_path.
+                        default=None,
+                        type=str)
+
+    parser.add_argument('-ofn', '--output-file-name',
+                        help='Filename for that will be created to store inspection results.'
+                             f'Default is "{EvaluationProcessNames.RESULTS_EXT.value}"',
+                        default=f'{EvaluationProcessNames.RESULTS_EXT.value}',
+                        type=str)
+
+    parser.add_argument(run_tool_arguments.FORMAT.value.short_name,
+                        run_tool_arguments.FORMAT.value.long_name,
+                        default=OutputFormat.JSON.value,
+                        choices=OutputFormat.values(),
+                        type=str,
+                        help=run_tool_arguments.FORMAT.value.help)
+
+
+def create_dataframe(config) -> Union[int, pd.DataFrame]:
```

Comment:
It is bad practice to return int and Dataframe. Please, just raise errors instead return ints